deployments/k8s/generate-from-helm

```diff
@@ -19,7 +19,8 @@ for f in $templates; do
     -x ${AGENT_CHART_DIR%/}/templates/$f \
     $AGENT_CHART_DIR | \
       sed -e 's/[[:space:]]*$//' | \
-      sed -e 's@apiVersion: apps/v1beta2@apiVersion: extensions/v1beta1@' | \
+      sed -e 's@apiVersion: apps/v1beta2@apiVersion: apps/v1@' | \
```

Comment:
Just fix the original resource templates to use apps/v1 and get rid of this sed stuff.  The only reason we had it to begin with was for when we had the Helm chart in the managed repository and they required all apiVersions to be the latest available.